CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 # Changelog
 Running changelog of releases since `2.0.0-rc.4`
 
+## v2.3.1
+
+### Updates
+- Support opaque private keys (#225) @aborrello
+- Improve oatuh2 servapp doc (#221)( @juanmatias
+- Removed omitempty from ApplicationGroupAssignment priority field (#218)( @bogdanprodan-okta
+- Added `WithHttpClientPtr`, `WithHttpClientPtr` (#214)( @bogdanprodan-okta
+- Remove omitempty of 'default_scope' field (#212)( @bogdanprodan-okta
+- Add exported OKTA_ACCESS_TOKEN constant (#213)( @bretterer
+- Adds Stalebot workflow (#210)( @bretterer
```

Comment:
nit: most lines have a stray `(` before author's handle. Can be removed. E.g (#210)`(`@bretterer

Reply:
I'll get that typo, thanks